x-pack/plugins/alerting/server/saved_objects/migrations.test.ts

```diff
@@ -1696,6 +1696,206 @@ describe('successful migrations', () => {
         },
       });
     });
+
+    test('security solution is migrated to saved object references if it has a "ruleAlertId"', () => {
```

Comment:
Can we add a test to ensure migration is skipped if alertTypeId is not `siem.notifications`?

Reply:
Yep, that's a good idea. Will add it and push up.